src/main/java/net/minecraftforge/common/world/ModifiableBiomeInfo.java

```diff
@@ -147,10 +147,21 @@ public ClimateSettingsBuilder getClimateSettings()
                 return climateSettings;
             }
 
+            /**
+             * {@return Builder for client special effects}
+             * 
+             * @deprecated Use {@link #getSpecialEffects()} as it provides read access. TODO remove this by 1.20
+             */
+            @Deprecated(forRemoval=true, since="1.19")
             public BiomeSpecialEffects.Builder getEffects()
             {
                 return effects;
             }
+            
+            public BiomeSpecialEffectsBuilder getSpecialEffects()
+            {
+            	return effects;
```

Comment:
Formatting: 4 spaces for indentation, not tabs
```suggestion
                return effects;
```